src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -2,71 +2,100 @@ import * as React from 'react';
+            this.setState({show: false});
+        } else {
+            const isIE11 = String(name) === "ie" && Number(version.slice(0,2)) === 11;
+            name = name.toLowerCase();
 
-  handleUnsupportedBrowsers(name:string, version: string) {
-      const sessionStorage = window.sessionStorage.browserError || false;
-      if (sessionStorage === true) {
-          this.setState({show: false});
-      } else {
-          const isIE11 = String(name) === "ie" && Number(version.slice(0,2)) === 11;
-          name = name.toLowerCase();
+            if (String(name) === 'unsupported') {
+                window.sessionStorage.browserError = true;
+                this.setState({show: true});
+            } else if (!(name === "chrome" || name === "firefox" || name === "edge" || name === "safari" || isIE11))  {
+                window.sessionStorage.browserError = true;
+                this.setState({show: true});
+            } else {
+                this.setState({show: false})
+                window.sessionStorage.browserError = true;
+            }
+        }
+    }
 
-          if (String(name) === 'unsupported') {
-              window.sessionStorage.browserError = true;
-              this.setState({show: true});
-          } else if (!(name === "chrome" || name === "firefox" || name === "edge" || name === "safari" || isIE11))  {
-              window.sessionStorage.browserError = true;
-              this.setState({show: true});
-          } else {
-              this.setState({show: false});
-              window.sessionStorage.browserError = true;
-          }
-      }
-  }
+    handleHideClick() {
+        this.setState({show: false});
+        window.localStorage.noShowModal = this.state.noShowModal;
+    }
 
-  public render() {
-      return (
-          <Modal show={this.state.show} onHide={() => this.setState({show: false})}>
-              <Modal.Header closeButton>
-                  <Modal.Title>Sorry, we do not support your browser version!</Modal.Title>
-               </Modal.Header>
-               <Modal.Body>
-                   <ul>
-                       <li>It looks like your using an {this.state.name === 'ie' ? ' old version of Internet Explorer' : 'unsupported browser'}.</li>
-                       <li>Please consider using IE11, Chrome, Safari, or Firefox.</li>
-                   </ul>
-               </Modal.Body>
-          </Modal>
+    public render() {
+        return (
+            <Modal show={this.state.show} onHide={this.handleHideClick}>
+                <Modal.Header closeButton>
+                    <Modal.Title>Sorry, we do not support your browser version!</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <ul>
+                        <li>It looks like your using an {this.state.name === 'ie' ? ' old version of Internet Explorer' : 'unsupported browser'}.</li>
+                        <li>Please consider using IE11, Chrome, Safari, or Firefox.</li>
```

Comment:
We should change this to "Latest version of Chrome, Safari, Firefox or Microsoft Edge" 

IE11 is actually two generations old at this point.